File: java/2차 1급 5_initial_code.java
// 다음과 같이 import를 사용할 수 있습니다.
import java.util.*;

class Solution {
    public int solution(int[] arr) {
        int answer = 1;

        for (int start = 0, end = 1; end < arr.length; end++) {
            if (arr[end - 1] < arr[end]) {
                answer = Math.max(end - start + 1, answer);
            } else {
                start = end;
            }
        }

        return answer;
    }

    // 아래는 테스트케이스 출력을 해보기 위한 main 메소드입니다.
    public static void main(String[] args) {
        Solution sol = new Solution();
        int[] arr = {3, 1, 2, 4, 5, 1, 2, 2, 3, 4};
        int ret = sol.solution(arr);

        // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
        System.out.println("solution 메소드의 반환 값은 " + ret + " 입니다.");
    }
}

File: python/2차 1급 5_initial_code.py
#다음과 같이 import를 사용할 수 있습니다.
#import math

def solution(arr):
    answer = 1
    start = 0

    for end in range(1, len(arr)):
        if arr[end - 1] < arr[end]:
            answer = max(end - start + 1, answer)
        else:
            start = end

    return answer

#아래는 테스트케이스 출력을 해보기 위한 코드입니다.
arr = [3, 1, 2, 4, 5, 1, 2, 2, 3, 4]
ret = solution(arr)


#[실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
print("solution 함수의 반환 값은", ret, "입니다.")